defaults/main.yml

```diff
@@ -13,23 +13,23 @@ midonet_gateway_external_nic: '{{ ansible_default_ipv4.interface }}'
 midonet_gateway_create_external_network: True
 midonet_gateway_external_network_name: 'ext-net'
 midonet_gateway_external_subnet_name: 'ext-subnet'
-midonet_gateway_external_subnet_pool_start: '{{ midonet_gateway_fip_net  | ipaddr("10") | ipaddr("address") }}'
-midonet_gateway_external_subnet_pool_end: '{{ midonet_gateway_fip_net | ipaddr("254") | ipaddr("address") }}'
-midonet_gateway_external_subnet_gw: '{{ midonet_gateway_fip_net  | ipaddr("1") | ipaddr("address") }}'
+midonet_gateway_external_subnet_pool_start: '{{ midonet_gateway_external_network_cidr  | ipaddr("10") | ipaddr("address") }}'
+midonet_gateway_external_subnet_pool_end: '{{ midonet_gateway_external_network_cidr | ipaddr("254") | ipaddr("address") }}'
+midonet_gateway_external_subnet_gw: '{{ midonet_gateway_external_network_cidr  | ipaddr("1") | ipaddr("address") }}'
 midonet_gateway_edge_router_name: 'edge-router'
 midonet_gateway_edge_network_name: 'net-edge1-gw1'
 midonet_gateway_edge_subnet_name: 'subnet-edge1-gw1'
 midonet_gateway_edge_port_name: 'port-edge1-gw1'
-midonet_gateway_fip_net: '200.200.200.0/24'
+midonet_gateway_external_network_cidr: '200.200.200.0/24'
```

Comment:
Well, this is not meant to give outside connectivity at all, is an example.
The operations guide uses this CIDR, and it's in the puppet modules also, so I would like to keep it as is.